curriculum/js1/8.js

```diff
@@ -5,8 +5,14 @@
  * @returns {number}
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (a, b, i=0, ans=0) => {
+  if(i>=a || i>=b){
```

Comment:
gcd of 5 and 7 is 1. But your soln returns 5

Reply:
fixed it!